package-lock.json

```diff
@@ -18110,10 +18124,11 @@
       }
     },
     "node_modules/typescript": {
-      "version": "5.4.2",
```

Comment:
Intriguingly, we were already at 5.4.2 (not 5.2.2 as you might think from package.json) because of api-extractor. It looks like that's using a pinned version of TS? It might be worth looking at upgrading it in the future too.